### ISSUE TITLE
Error while getting certificate exp. dates 

### ISSUE THREAD

**Issue by william-sy** [xolphin/xolphin-api-python#4]
Hi there seems to be a slight issue in the code:
```
import xolphin
client = xolphin.Client('', '')

certificates = client.certificate().all()
for certificate in certificates:
    print(certificate.id, certificate.isExpired())
```
Results in: 

```
Traceback (most recent call last):
  File "./test.py", line 8, in <module>
    certificates = client.certificate().all()
  File "/usr/local/lib/python2.7/dist-packages/xolphin/endpoint/certificate.py", line 18, in all
    result = Requests(self.client.get('certificates', {'page': result.page + 1}))
  File "/usr/local/lib/python2.7/dist-packages/xolphin/responses/requests.py", line 11, in __init__
    for request in self._embedded['requests']:
KeyError: 'requests'
```

This is on ubuntu 16 and 14

**Comment by DDaveF** [comment]
If you want it to work right away, edit this file:
xolphin/endpoint/certificate.py

Modify line 18 and replace "Requests" with "Certificates":
``
result = Certificates(self.client.get('certificates', {'page': result.page + 1}))
``

**Comment by XolphinIvo** [comment]
Solved in master, created new tag (1.6.0).
